doc/doxygen/src/porting-boards.md

```diff
@@ -334,3 +337,10 @@ Some scripts and tools available to ease `BOARD` porting and testing:
 
   - Run `dist/tools/compile_and_test_for_board/compile_and_test_for_board.py . <board> --with-test-only`
     to run all automated tests on the new board.
+
+# Further reference                                         {#further-reference}
+
+- [In her blog][martines-blog], Martine Lenders documented her approach of
+  porting the @ref boards_feather-nrf52840 in February 2020.
```

Comment:
BTW feel free to provide more examples here that I could integrate here. Feels a bit pretentious to only cite my own blog here ;-).

Reply:
I think it is great!

Reply:
I did a porting guide for [EFM32HG](https://hackmd.io/njFHwQ33SNS3sQKAkLkNtQ). Could be useful for similar ones as well. The porting guide does not include documentation on the generator tools used by @basilfx though.

Reply:
why not adding this documentation as a concrete example for porting to the doxygen?

Reply:
@akshaim I can link your guide. How would you prefer it to be cited here?

Reply:
You can have it as an additional reference for quick help. I was trying to document the process of board port for other platforms but it seems like there is no easy way to do it. The above example I cited is an easy "dirty" way to port the CPU but not the "recommended" method as different maintainers use different processes.  Which do you think makes sense to document the easy method or recommended method?

Reply:
Added.

Reply:
>  Which do you think makes sense to document the easy method or recommended method?

I never ported a CPU, so I can't really tell what methods are there and which are preferred by whom. Me in your place would go for what you think is the best method first and then ask for input by other maintainers in the PR.

Reply:
Okay. That sounds good to me  :)